site/documentation/content/admin-guide/jdbc-device-registry-config.md

```diff
@@ -47,10 +47,11 @@ and availability.
 | `HONO_REGISTRY_JDBC_MANAGEMENT_PASSWORD`         <br> `--hono.registry.jdbc.management.password`          | no  | -    | The password used to access the database. |
 | `HONO_REGISTRY_JDBC_MANAGEMENT_MAXIMUMPOOLSIZE`  <br> `--hono.registry.jdbc.management.maximumPoolSize`   | no  | Depends on the connection pool implementation. `15` for C3P0. | The maximum size of the connection pool. |
 | `HONO_REGISTRY_JDBC_MANAGEMENT_TABLENAME`        <br> `--hono.registry.jdbc.management`                   | no  | -    | The name of the table the datastore uses. If the datastore requires multiple tables, this is the prefix. |
-| `HONO_REGISTRY_SVC_TASKEXECUTORQUEUESIZE`        <br> `--hono.registry.svc.taskExecutorQueueSize`         | no  | `1024` | The size of the executor queue for hashing passwords. |
 | `HONO_REGISTRY_SVC_CREDENTIALSTTL`               <br> `--hono.registry.svc.credentialsTtl`                | no  | `1m` | The TTL for credentials responses. |
-| `HONO_REGISTRY_SVC_REGISTRATIONTTL`              <br> `--hono.registry.svc.registrationTtl`               | no  | `1m` | The TTL for registrations responses. |
 | `HONO_REGISTRY_SVC_MAXBCRYPTITERATIONS`          <br> `--hono.registry.svc.maxBcryptIterations`           | no  | `10` | The maximum number of allowed bcrypt iterations. |
+| `HONO_REGISTRY_SVC_REGISTRATIONTTL`              <br> `--hono.registry.svc.registrationTtl`               | no  | `1m` | The TTL for registrations responses. |
+| `HONO_REGISTRY_SVC_TASKEXECUTORQUEUESIZE`        <br> `--hono.registry.svc.taskExecutorQueueSize`         | no  | `1024` | The size of the executor queue for hashing passwords. |
+| `HONO_REGISTRY_SVC_USERNAMEPATTERN`              <br> `--hono.registry.svc.usernamePattern`               | no | `^[a-zA-Z0-9-_=\\.]+$` | The regular expression to use for validating authentication identifiers (user names) of hashed-password credentials. |
```

Comment:
It is striking that `username` instead of `authId` is used here. Any particular reason for that? It seems to me, the `authentication id` term is more commonly used in the Credentials API.

Reply:
Yes, that is true. But the config property only affects the auth-id of hashed-password credentials where the auth-id is usually referred to as a *username*. Another alternative would be to call the property `HONO_REGISTRY_SVC_HASHEDPASSWORDAUTHIDPATTERN`. Personally, I prefer the current name ...

Reply:
Ok, I see.